### ISSUE TITLE
Support for Blood of Luclin

### ISSUE THREAD

**Issue by DragonMrak** [Drumstix42/DrumsUI#1]
Hello,

What about new expansion pack, any plans to have an UI update? As far as I noticed, not much to be changed actually...

**Comment by Drumstix42** [comment]
@DragonMrak I'm not (personally) supporting the UI actively anymore. I mainly threw the code onto Github to have it preserved (just in case EQ2Interface became unavailable).

There is another user, Blackhood on EQ2i, who has been maintaining it on my behalf who may very well update any required changes. Also some other community members have pitched in changes along the way. So I would keep an eye on the forums/downloads over there.

When free time arises, I try to get the latest changes onto the patcher so that the auto-updated still works!

**Comment by DragonMrak** [comment]
Yes, there is an update for DrumsUI on eq2interface by Blackhood which makes it compatible with Chaos Descending, and which is not on autoupdater as of yet. Since I do not have neither eq2interface nor eq2i accounts, but I do have GitHub one, it was easiest way for me to write here :-)

I'll try to find Blackhood before the expansion hits live servers. Meanwhile, can I make some updates on this repository?

**Comment by DragonMrak** [comment]
Whoops, I see that I have missed the whole forum on eq2interface somehow :-)

**Comment by Drumstix42** [comment]
Yes, there is a sub-forum over there. I would say making PRs here would be okay, but I would need to first make sure the repo is up-to-date (or someone else could make that PR first).
